pinot-compatibility-verifier/src/main/java/org/apache/pinot/compat/QueryOp.java

```diff
@@ -46,6 +46,9 @@
   private static final String COMMENT_DELIMITER = "#";
   private String _queryFileName;
   private String _expectedResultsFileName;
+  private Boolean _shouldCompareNumEntriesScannedInFilter = false;
```

Comment:
nit: why use Boolean instead of boolean here?

Reply:
changed to boolean